### PR TITLE
add code from the task

### PR DESCRIPTION
**Pull request by v1tal4ik**

### CODE REVIEW

File: hw1/v1tal4ik.test.js [hunk 1]
```diff
@@ -0,0 +1,314 @@
+const { repeatString } = require('./v1tal4ik');
```
Comment (vickierose) [inline hw1/v1tal4ik.test.js:1]: `import` didn't work?

File: hw1/v1tal4ik.js [hunk 2]
```diff
@@ -0,0 +1,173 @@
+// 1.Write a Javascript program to create a new string which is n copies of a given string where n is a non-negative integer.
+
+function repeatString(str = '', value = 0) {
```
Comment (vickierose) [inline hw1/v1tal4ik.js:3]: nice

File: hw1/v1tal4ik.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  return basicStr;
+}
+
+const repeatStr = repeatString('Hi Tomas!', 3);
+console.log({ repeatStr });
+
+
+// 2.Write a Javascript program to check if a string starts with "if".
+
+function checkIfStartsWithIf(str = '') {
```
Comment (vickierose) [inline hw1/v1tal4ik.js:23]: there is a simpler solution but this one demostrates that you know what regExp is 😉

File: hw1/v1tal4ik.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+}
+
+const example = 'if i were sultan, I will have a lots of wife';
+const isStart = checkIfStartsWithIf(example);
+console.log({ isStart });
+
+
+// 3.Write a Javascript program to check two non-negative integer values and return true if they have the same last digit
+
+function checkLastDigit(num1 = 0, num2 = 0) {
```
Comment (vickierose) [inline hw1/v1tal4ik.js:45]: nice and very detailed

File: hw1/v1tal4ik.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+function getSomeInfoOfNumber(number = 0) {
+  if (typeof (number) !== 'number') {
+    console.log('Error: type of argument must be Number');
+    return { message: 'Error: type of argument must be Number' };
+  }
+  if (number <= 0) {
+    console.log('Error: argument must be positive number');
+    return { message: 'Error: argument must be positive number' };
+  }
+  if (number % 1 !== 0) {
```
Comment (vickierose) [inline hw1/v1tal4ik.js:82]: like this check!

File: hw1/v1tal4ik.js [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+/*
+
+5. What would be the result of next statements:
+[1, 2, 3] + {} || 1,2,3[object Object]
+1 + "3" - 2    || 11
+1 + ("3" - 2)  || 2
+
+What would a, b and c equal?
+  let a = 2;    || 2
```
Comment (vickierose) [inline hw1/v1tal4ik.js:126]: nope. paste the code to browser console and then log these variables

File: hw1/v1tal4ik.js [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+*/
+
+
+// 6. Get type of nn in a lower case format:
+
+const nn = new Promise((res, rej) => {
+  setTimeout(() => res('done'), 1000);
+});
+
+const getDataType = (obj = 'empty') => {
```
Comment (vickierose) [inline hw1/v1tal4ik.js:139]: very detailed.
But what will happen if I want to pass there a string 'empty'?

**Review by vickierose — COMMENTED** [review]
Impressive, especially tests. Still, take a look at the comments

File: hw1/v1tal4ik.test.js (hunk 1 above)
Comment (v1tal4ik) [reply hw1/v1tal4ik.test.js:1]: work, but I haven't got time to install package for it.

**Pull request merged**